src/redux/reducer/jobReducer.tsx

```diff
@@ -59,13 +60,48 @@ export const paginationApi = (currentPage: number) => {
   };
 };
 
-export const searchJobApi = (searchValue: string) => {
+export const searchJobByTitleApi = (searchValue: string) => {
```

Comment:
Please use one search handler function instead of three

Reply:
Fixed